wscript

```diff
@@ -421,6 +421,11 @@ audio_output_features = [
         'desc': 'SDL2 audio output',
         'deps': 'sdl2',
         'func': check_true,
+    }, {
+        'name': '--oss-audio',
+        'desc': 'OSS',
+        'func': check_cc(header_name='sys/soundcard.h'),
```

Comment:
Without excluding [OSSv3 emulation on Linux](https://www.kernel.org/doc/html/latest/sound/designs/oss-emulation.html) it's going to block further cleanup i.e., see [TravisCI build logs](https://travis-ci.org/github/mpv-player/mpv/pull_requests).

Reply:
I hope last change will solve it.